root/static/scripts/common/components/EntityLink.js

```diff
@@ -145,7 +145,9 @@ const EntityLink = (props = {}) => {
 
     if (nameVariation) {
       if (hover) {
-        hover = l('{name} – {additional_info}', {name: entity.name, additional_info: hover});
+        if !(entity.name == entity.sortname) {
```

Comment:
A `!` outside the parens is not valid JavaScript. :P

Try `(entity.name !== entity.sortname)` instead.